src/Commands/Backup/Automatic/EnableCommand.php

```diff
@@ -15,19 +15,19 @@ class EnableCommand extends TerminusCommand implements SiteAwareInterface
     use SiteAwareTrait;
 
     /**
-     * Set up a week-long TTL backups to be made daily and a month-long TTL to be made weekly
+     * Enables automatic backups and retention period.
      *
      * @authorize
      *
      * @command backup:automatic:enable
      *
-     * @param string $site_env Site & environment to set the schedule of, in the format `site-name.env`.
+     * @param string $site_env Site & environment in the format `site-name.env`
      * @option string $day Day of the week to make the month-long backup in any format recognized by PHP strtotime
      *
      * @usage terminus backup:automatic:enable <site>.<env>
-     *     Sets backups to occur at a random hour, with month-long TTL backup made on a random day
+     *     Enables automatic daily backups to be retained for 7 days.
```

Comment:
The text on this command's usage doesn't include <site> and <env> like the others do.